### DISCUSSION TITLE
Jellyfin

### DISCUSSION THREAD

**Discussion by LaMpiR** [swizzin/swizzin#589] [Q&A + Help]
Hi.

Swizzin uptodate. Wanted to try out the jellyfin.

Installed through "box install jellyfin" as a root. Installation was without any errors. Went to the panel, clicked o Jellyfin and got the error "Error processing request." from url */jellyfin.

Went to *:8920 but no ssl and it works.

Is there anyway to install the ssl for the *:8920 and to change the url in the cotrol panel menu?

No LSB modules are available.
Distributor ID: Ubuntu
Description: Ubuntu 20.04.2 LTS
Release: 20.04
Codename: focal

**Comment by LaMpiR** [comment]
I have corrected my original post.

**Comment by flying-sausages** [comment]
Heya, sorry but I haven't seen this. Is this still happening? If so, please make an issue or come to the Discord.
